### PR TITLE
Improve CONTRIBUTING file

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [x] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Updated the Contributing file with better formatting and a bit more info about topics like how to make proper Pull requests.

Closes #

[contributing]: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Review by abhijoshi2k — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
@Andre601 I have requested a change

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
-This project and everyone participating in it is governed by the [CODE OF CONDUCT](https://github.com/Readme-Workflows/recent-activity/blob/main/CODE_OF_CONDUCT.md), By participating, you are expected to uphold this code. Please report unacceptable behavior to baalkrshna@gmail.com or abhijoshi2k@gmail.com.
+## General Support
 
-## Asking Questions
-
-Please read [SUPPORT.md](https://github.com/Readme-Workflows/recent-activity/blob/main/.github/SUPPORT.md) for knowing where to ask questions
-
-Don't use GitHub issues for asking questions, we use GitHub issues for bug reports and feature requests.
+Please read the [SUPPORT.md file][support] for how to get support with Recent Activity.  
+Do not use issues to ask questions. Issues are reserved for Bug reports and Feature requests. Any issue for asking question will be closed or (more likely) changed into a [Discussion].
```
[the remark this answers is not in the document]
Comment (PuneetGopinath) [reply .github/CONTRIBUTING.md:25]: Actually how did you give the option of commit suggestion, I got that option, before when you reviewed some of my PRs.

[the remark this answers is not in the document]
Comment (Andre601) [reply .github/CONTRIBUTING.md:25]: There is this button where you set the change you request.
image

Alternatively can you also just use a code block and set "suggestion" as the code highlighting.

**Review by PuneetGopinath — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
Or else I itself will do it, no probs

**Pull request merged**